rootfs/bin/create_bucket

```diff
@@ -23,9 +24,20 @@ region = os.getenv('AWS_REGION')
 
 if os.getenv('DATABASE_STORAGE') == "s3":
     conn = boto.s3.connect_to_region(region)
-
     if not bucket_exists(conn, bucket_name):
-        conn.create_bucket(bucket_name, location=region)
+        try:
+            conn.create_bucket(bucket_name, location=region)
+        except S3CreateError as err:
+            # try again in the default S3 region before giving up
+            default_region = 'us-east-1'
+            if region != default_region:
+                print('Failed to create bucket in {}, trying default region {}...'.format(
+                    region, default_region))
+                conn = boto.s3.connect_to_region(default_region)
+                if not bucket_exists(conn, bucket_name):
+                    conn.create_bucket(bucket_name, location=default_region)
+                # TODO: if we got here, somehow we need to propagate "us-east-1"
```

Comment:
An easy way we can accomplish this is by writing `https+path://s3.amazonaws.com:443` to $WALE_ENVDIR/WALE_S3_ENDPOINT.

See https://github.com/deis/postgres/blob/6863e0b398b6e254d8639115123f3d3177002db1/rootfs/docker-entrypoint-initdb.d/001_setup_envdir.sh#L11-L19 for inspiration/background